framework/include/ble_manager/ble_client.h

```diff
@@ -0,0 +1,293 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/**
+ * @defgroup BLE_Client
+ * @ingroup BLE_Client
+ * @brief Provides APIs for BLE Client
+ * @{
+ */
+
+/**
+ * @file ble_manager/ble_client.h
+ * @brief Provides APIs for BLE Client
+ */
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "ble_common.h"
+
+typedef enum {
+	BLE_CLIENT_SCAN_STARTED,
+	BLE_CLIENT_SCAN_STOPPED,
+} ble_client_scan_state;
```

Comment:
Some of enum variables have the postfix of '_e' but some of them don't.
Why don't you keep consistency on these naming rules?

Reply:
Actually, there was no rule for it.
I will remove '_e' in BLE manager

Reply:
Adding '_e' in enum type is mendatory in TizenRT.
I will add '_e' in every enums